trustfall_testbin/src/main.rs

```diff
@@ -1,6 +1,7 @@
 #![forbid(unsafe_code)]
 #![forbid(unused_lifetimes)]
 
+use anyhow::Context;
```

Comment:
```suggestion
use anyhow::Context as _;
```
This is a cute little trick: it imports the trait (making its methods visible) but without importing the name `Context`.

Reply:
TIL